syncpack.config.cjs

```diff
@@ -10,15 +10,11 @@ module.exports = {
 	workspace: false,
```

Comment:
Just to be clear re: my previous comment, this line won't do anything and will need updating if you want its previous behaviour.

Reply:
Thanks! Hadn't pushed my latest changes. Should be removed now.